website/integrations/services/owncloud/index.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Owncloud
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Owncloud
+
+> Owncloud is a free and open-source software project for content collaboration and sharing and syncing of files
+>
+> -- https://en.wikipedia.org/wiki/Owncloud
+
+:::note
+This guide focuses on Owncloud installations deployed using Docker. If you have deployed it using a different mechanism,
+there may be some differences in the process.
+:::
+
+## Authentication
+
+There exist Owncloud plugins to support multiple authentication flavors, including:
+
+-   [SAML/SSO](https://doc.owncloud.com/server/10.15/admin_manual/enterprise/user_management/user_auth_shibboleth.html)
+-   [OAuth2](https://doc.owncloud.com/server/10.15/admin_manual/configuration/user/user_oauth2.html)
+-   [OpenID Connect](https://doc.owncloud.com/server/10.15/admin_manual/configuration/user/oidc/oidc.html)
+
+This guide will focus on OpenID Connect (OIDC)
+
+The following placeholders will be used in this document:
+
+-   `owncloud.company`: FQDN of the Owncloud install.
+-   `authentik.company`: FQDN of the authentik install.
+
+### OpenID Connect (OIDC)
+
+OIDC is supported in Owncloud through the use of the official [OIDC plugin](https://github.com/owncloud/openidconnect).
+
+#### authentik
+
+Ensure that you have a Signing Key available in authentik for use with these providers. Create one if necessary
+under _System_ > _Certificates_.
```

Comment:
I broke the lines (here and below) for readability in the editor. I can join everything back to a single line if you prefer

Reply:
doing it here doesn't change anything, it will produce exactly the same formatting in the browser. A single line break doesn't change formatting at all, it just makes it easier to read in an editor with line wrapping disabled.